MaxText/tests/tokenizer_test.py

```diff
@@ -61,14 +61,43 @@ def tearDownClass(cls):
   @pytest.mark.tpu
   def test_tokenize(self):
     text = 'This is a test'
-    self.assertTrue(np.array_equal(self.source_tokenizer.tokenize(text).numpy()[1:-1],
-                                    self.test_tokenizer.tokenize(text).numpy()))
+    self.assertTrue(np.array_equal(self.source_tokenizer.encode(text).numpy(),
+                                    self.test_tokenizer.encode(text).numpy()))
 
   @pytest.mark.tpu
   def test_detokenize(self):
     tokens = [66, 12, 10, 698]
-    self.assertEqual(np.asarray(self.source_tokenizer.detokenize(tokens)), 
-                     np.asarray(self.test_tokenizer.detokenize(tokens)))
+    self.assertEqual(np.asarray(self.source_tokenizer.decode_str(tokens)), 
+                     np.asarray(self.test_tokenizer.decode_str(tokens)))
+    
+
+class TikTokenTest(unittest.TestCase):
+  """Tests for train_tokenizer.py"""
+
+  @classmethod
+  def setUpClass(cls):
+    dataset_name = "c4/en:3.0.1"
+    dataset_path = "gs://maxtext-dataset"
+    cls.source_tokenizer = input_pipeline_interface.get_tokenizer("../assets/tokenizer_llama3.tiktoken", add_bos=False, add_eos=False)
```

Comment:
to clarify: if we are using `tiktoken` from `engine`, are we still using `../assets/tokenizer_llama3.tiktoken`?

Reply:
Both maxengine.py and the tokenizer.py codepaths for getting the tokenizer use `assets/tokenizer_llama3.tiktoken`.

Reply:
SG - thanks Mohit